### PR TITLE
[bitnami/postgresql-ha] add loadBalancingOnWrite behavior configuration property

### PR DESCRIPTION
**Pull request by dani8art**
**Description of the change**

Add a new property to configure the load balancing behavior. We are able to do it since https://github.com/bitnami/bitnami-docker-pgpool/pull/17

**Benefits**

Users can select this behavior

**Possible drawbacks**

N/A

**Applicable issues**

  - fixes #

**Additional information**

N/A

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/postgresql-ha/values-production.yaml
Comment (miguelaeh) [inline bitnami/postgresql-ha/values-production.yaml:239]: Do not remove the empty lines

File: bitnami/postgresql-ha/templates/pgpool/deployment.yaml
Comment (miguelaeh) [inline bitnami/postgresql-ha/templates/pgpool/deployment.yaml:69]: The env var is called `PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE` and the parameter `pgpool.disableLoadBalancingOnWrite`, that seems to be contradictory

File: bitnami/postgresql-ha/README.md
Comment (miguelaeh) [inline bitnami/postgresql-ha/README.md:138]: `balance` -> `balancer`

File: bitnami/postgresql-ha/values-production.yaml
Comment (dani8art) [reply bitnami/postgresql-ha/values-production.yaml:239]: it was done by my linter automatically, fixing it!

File: bitnami/postgresql-ha/templates/pgpool/deployment.yaml
Comment (dani8art) [reply bitnami/postgresql-ha/templates/pgpool/deployment.yaml:69]: Yes, your right! changing it!

**Comment by miguelaeh** [comment]
Wait for the CI/CD to update the tags

**Review by carrodher — APPROVED** [review]
Are some changes required in the Docker image to support the new env. variable? If so, LGTM if the image was already released and included in the current chart version, in another case we should ensure the image tag is updated in the PR

**Comment by nc-ruth** [comment]
Hi @dani8art @carrodher @miguelaeh, what's the status of this PR?

**Comment by javsalgar** [comment]
It seems that it got lost. Closing this one and creating a new one

https://github.com/bitnami/charts/pull/7134

**Pull request closed without merging**
